wdn/templates_4.1/includes/scriptsandstyles_speedy.shtml

```diff
@@ -0,0 +1,42 @@
+<noscript>
+    <link rel="stylesheet" href="https://cloud.typography.com/7717652/616662/css/fonts.css" />
+    <link rel="stylesheet" href="https://unlcms.unl.edu/wdn/templates_4.1/css/all.css?dep=$DEP_VERSION$" />
+</noscript>
+<style id="wdn-critical-styles">
+    <!--#include virtual="/wdn/templates_4.1/css/critical.css" -->
```

Comment:
The includes should remain static, server tech independent files. For the reasons mentioned above about how our sync process works, I would not put a SSI statement in this file, otherwise no other server technology (php, jsp, etc) will be able to use it. 

Another idea for this might be to use add a set in the build process that builds this include file from a prefix file, the built css file, and a suffix file.